db/migrate/20180926205855_generate_web_channel_access_token_when_missing.rb

```diff
@@ -0,0 +1,17 @@
+class GenerateWebChannelAccessTokenWhenMissing < ActiveRecord::Migration[5.1]
+  def up
+    Bot.all.each do |bot|
```

Comment:
There is no need to iterate through all bots here, only all channels.

And it's a good practice to never use a model in a migration. It may be run after that model has been deleted. If it's easier to write it if the model is defined, redefine it in the migration itself so it's always available.

And you left some puts over there too 😄

Also, I'm not entirely sure, but I think a migration without down may crash when trying to rollback it. If that's the case, an empty method should do just fine.

Reply:
About there is no need to iterate through all bots here, totally. I'll change that.
About the good practice of avoiding the use of a model in a migration, I think I haven't totally get it. I've checked here: https://guides.rubyonrails.org/v3.2/migrations.html#using-models-in-your-migrations. At the moment, it does feel right to me using the model here, but I can be wrong (as ever!) and I do like to follow good practices, so 🙏 let's dig a little more into this in person.
About the rescue block (and the puts), I was inspired here: https://github.com/instedd/aida-ui/commit/9ee880e61c79c1009768f4dfbc471c7234cbba58#diff-5f158516cc5642ad2262667dd930c9d3R13 🤷‍♂️  Shouldn't we remove the rescue block entirely and let the migrations crash?